mesh/storage.c

```diff
@@ -92,8 +92,10 @@ static bool read_net_keys_cb(uint16_t idx, uint8_t *key, uint8_t *new_key,
 	if (!net)
 		return false;
 
-	if (mesh_net_add_key(net, false, idx, key) != MESH_STATUS_SUCCESS)
+	if (mesh_net_add_key(net, false, idx, key) != MESH_STATUS_SUCCESS) {
+		l_debug("cannot add net key");
```

Comment:
I'd say this is an error (or at least a warning), not debug.

Reply:
Done